File: 485. Max Consecutive Ones/solution.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        m = 0
        for num in nums:
            if num == 1:
                count += 1
                if count > m:
                    m = count
            else:
                count = 0
        return m

File: 485. Max Consecutive Ones/solution.go
func findMaxConsecutiveOnes(nums []int) int {
	result := 0
	consecutiveCnt := 0
	for _, num := range nums {
		if num == 1 {
			consecutiveCnt++
		} else {
			consecutiveCnt = 0
		}
		if result < consecutiveCnt {
			result = consecutiveCnt
		}
	}
	return result
}